pandas/core/frame.py

```diff
@@ -4341,27 +4341,31 @@ def last_valid_index(self):
 
     def pivot(self, index=None, columns=None, values=None):
         """
+        Return reshaped DataFrame summarized by given index / column values.
+
         Reshape data (produce a "pivot" table) based on column values. Uses
         unique values from index / columns to form axes of the resulting
```

Comment:
can you add single backtick around index and column ? (to reference them as parameters). 

I would maybe also say "from specified index / columns" to make it clear it is not the existing index / columns